src/Oceananigans.jl

```diff
@@ -132,6 +132,10 @@ function __init__()
     end
 end
 
+# Supported types for the Advection module. 
+# To support new floating point types, add them here
+const supported_float_types = [Float32, Float64]

```

Comment:
I think this should be a tuple rather than array. Otherwise users can change it eg

```julia
push!(Oceananigans.supported_float_types, Float16)
```

but of course this does nothing.